common/web/lm-worker/build-polyfiller.js

```diff
@@ -111,7 +176,19 @@ console.log("Pass 2:  Output intermediate state and perform minification");
 fullWorkerConcatenation.script = fullWorkerConcatenation.script.substring(0, fullWorkerConcatenation.script.lastIndexOf('//# sourceMappingURL'));
 fullWorkerConcatenation.script += `//# sourceMappingURL=${fullWorkerConcatenation.scriptFilename}.map`;
 
-fs.writeFileSync(`build/lib/${fullWorkerConcatenation.scriptFilename}`, fullWorkerConcatenation.script);
+fs.writeFileSync(`${fullWorkerConcatenation.scriptFilename}`, fullWorkerConcatenation.script);
 if(fullWorkerConcatenation.sourcemapJSON) {
-  fs.writeFileSync(`build/lib/${fullWorkerConcatenation.scriptFilename}.map`, convertSourceMap.fromObject(fullWorkerConcatenation.sourcemapJSON).toJSON());
-}
\ No newline at end of file
+  fs.writeFileSync(`${fullWorkerConcatenation.scriptFilename}.map`, convertSourceMap.fromObject(fullWorkerConcatenation.sourcemapJSON).toJSON());
+}
+
+await esbuild.build({
+  entryPoints: [destFile],
+  sourcemap: 'external',
+  sourcesContent: true,
+  minify: true,
+  // Do NOT enable - will break under Android 5.0 / Chrome 35 environments, likely through Chrome 42.
+  // https://caniuse.com/mdn-javascript_builtins_function_name_configurable_true
```

Comment:
This looks like it might be flipped in 18.0? Do we need an issue to track this?

Reply:
Assuming I'm properly understanding you here, we will not need an issue to track this.  From #10257:

https://github.com/keymanapp/keyman/blob/dc7bfc1344793cd30190ccb87a68272f2cdb9f3d/common/web/es-bundling/src/configuration.mts#L24-L35

The ES6 target's configuration will re-enable this.

Reply:
Will we ever want `keepNames: false`?

Reply:
It's not viable on Android 5.0 devices, so yes.